src/shared/Z.EF.Plus.BatchUpdate.Shared/BatchUpdateVisitor.cs

```diff
@@ -41,5 +41,15 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
 
             return base.VisitMethodCall(node);
         }
+
+        protected override Expression VisitMember(MemberExpression node)
+        {
+            if (node.Expression?.GetType().Name == "PropertyExpression")
```

Comment:
node.Expression is PropertyExpression

Reply:
Well, I want to be precise with the expression type. Just want to cover a case when there is access to a member through a navigation property because eventually it produces sql join statement.
For instance,
```
Expression<Func<TEntity, bool>> t = e => e.ChildEntity.SomeId == 456456;
```